dotnet-web/pipeline.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+  - label: ":package: Restore dependencies"
```

Comment:
Maybe the `:dotnet:` emoji is the way to go? 

https://github.com/buildkite/emojis